gradle.properties

```diff
@@ -36,3 +36,5 @@ kotlin.incremental=true
 kotlin.incremental.js=true
 
 org.gradle.caching=true
+
+jvmOnly=true
```

Comment:
I don't think we need this change commited. Could you remove it?

Reply:
fixed. Sorry about that.